manifests/profile/pdf_stack.pp

```diff
@@ -37,6 +37,11 @@
   package { $fonts:
     ensure => present,
   }
+  
+  # This must be installed before the eventmachine gem so we get https support
```

Comment:
Does this need a `before` metaparameter so it's installed before the eventmachine gem, or does eventmachine not go through a native build process at installation?